scripts/west_commands/sbom/README.rst

```diff
@@ -367,3 +375,10 @@ There are two additional methods for improving the correctness of the above algo
      :class: highlight
 
      -d build_directory *target.elf*:*file.map*
+
+Integration with the GN meta-build system
+=========================================
+
+The ``ncs-sbom`` reads a list of all commands needed to build provided targets.
+If there is a ``gn gen`` command, the script enters ``gn`` build directory and tries to extract files from it using the same method as described above.
```

Comment:
```suggestion
If there is a ``gn gen`` command, the script enters the :file:`gn` build directory and tries to extract files from it using the same method as described earlier.
```

Reply:
I think it's supposed to be "the script enters the build directory used by GN and...". `gn`is not the name of the directory.

Reply:
Maybe it can be something like that?
```
If there is a ``gn gen`` command, the script enters the command's build
directory and tries to extract files from it using the same method
as described earlier.
```

Reply:
@peknis, what do you think about above sentence?